baseplate/clients/cassandra.py

```diff
@@ -274,17 +321,29 @@ def execute(
         query: Query,
         parameters: Optional[Parameters] = None,
         timeout: Union[float, object] = _NOT_SET,
+        query_name: Optional[str] = None,
         **kwargs: Any,
     ) -> Any:
-        return self.execute_async(query, parameters, timeout, **kwargs).result()
+        return self.execute_async(
+            query, parameters=parameters, timeout=timeout, query_name=query_name, **kwargs
+        ).result()
 
     def execute_async(
         self,
         query: Query,
         parameters: Optional[Parameters] = None,
         timeout: Union[float, object] = _NOT_SET,
+        query_name: Optional[str] = None,
         **kwargs: Any,
     ) -> ResponseFuture:
+        prom_labels = CassandraPrometheusLabels(
+            cassandra_contact_points=",".join(self.cluster.contact_points),
```

Comment:
🔕 is this ordering deterministic? Perhaps they should be alphabetically ordered so the same set doesn't introduce more values when they're ordered differently.

Reply:
Done just in case :-)